### PR TITLE
freewheel: support int and string for zoneId

### PR DESCRIPTION
**Pull request by mwang-sticky**

### CODE REVIEW

File: openrtb_ext/imp_freewheelssp.go [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package openrtb_ext
 
+import "encoding/json"
+
 type ImpExtFreewheelSSP struct {
-	ZoneId int `json:"zoneId"`
+	ZoneId json.Number `json:"zoneId"`
```
Comment (gargcreation1992) [inline openrtb_ext/imp_freewheelssp.go:6]: you could use `jsonutil.StringInt` for [this change. ](https://github.com/prebid/prebid-server/blob/a1f48b54af8e831927a0f6838f56493fa03efb08/util/jsonutil/stringInt.go#L7-L8)

File: static/bidder-params/freewheelssp.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
   "properties": {
     "zoneId": {
-      "type": "integer",
+      "type": ["integer", "string"],
```
Comment (gargcreation1992) [inline static/bidder-params/freewheelssp.json:9]: Please add test cases for this change.

File: static/bidder-params/freewheel-ssp.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
   "properties": {
     "zoneId": {
-      "type": "integer",
+      "type": ["integer", "string"],
```
Comment (gargcreation1992) [inline static/bidder-params/freewheel-ssp.json:9]: Please add test case for this change.

**Comment by mwang-sticky** [comment]
thanks for your information and I add related ut, thanks

**Review by gargcreation1992 — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**